### DISCUSSION TITLE
About language-switching

### DISCUSSION THREAD

**Discussion by NisonChrist** [SydneyOwl/senhaix-freq-writer-enhanced#20] [Ideas]
I just found out that there's a lot of handling in the code that involves language-switching， such as [language-switching1](https://github.com/SydneyOwl/senhaix-freq-writer-enhanced/blob/372b89c76d2879be39289be5d23b77c6004b5140/GT12/SHX_GT12_CPS/FormAbout.cs#L28C5-L44C6)  and [language-switching2](https://github.com/SydneyOwl/senhaix-freq-writer-enhanced/blob/372b89c76d2879be39289be5d23b77c6004b5140/GT12/SHX_GT12_CPS/AppData.cs#L22C5-L34C7). That's struggling. Please don't use hard coding to handle language-switching but use JSON instead. We need the code to be more robust, flexible, and easy to maintain.

**Comment by SydneyOwl** [comment]
The original code has hardcoded language settings within the code itself, so implementing an alternative approach would require modifications in numerous places, constituting a substantial undertaking. Furthermore, given that the software is primarily intended for domestic users, there's currently no consideration for adopting a more advanced language switching mechanism. Thank you.

**Comment by SydneyOwl** [comment]
However I would be immensely grateful if anyone could assist in implementing such a feature.

**Comment by SydneyOwl** [comment]
Now a work on internationalization (i18n) is currently underway! 
The work is currently in its initial stages, and most of the text is still being translated. See commit c5a49fb81b9c2af9b4e31235f10147f8e5fa3ee5 for more!

**Reply by NisonChrist** [reply]
Happy to hear that！Looks good to me！Nice job！
